### PR TITLE
Simplify Lockbox introduction strings

### LINKED ISSUE #3715: Link text is in English in localized page for Lockbox
This is how it currently looks for Italian (bad…)
https://testpilot-l10n.dev.mozaws.net/experiments/firefox-lockbox

schermata 2018-07-06 alle 12 34 02

cc @devinreams

### PR DESCRIPTION
**Pull request by devinreams**
Fixes #3715 

The intention here was to "work around" the rendering issue (English text is displayed in the anchor tag text on non-English pages) by simply replacing the tag text with actual URLs.

screen shot 2018-08-07 at 3 53 29 pm

It's not pretty, but it's seemingly less bad than having "here" show up in the middle of a translation.

### CODE REVIEW

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3784/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/edb6ddbb1e89db8e451d1cd79f2611136c10cf5f/

File: locales/en-US/experiments.ftl [hunk 1]
```diff
@@ -75,7 +75,7 @@ containersContributors7Title = Firefox UX
...
+firefoxlockboxIntroduction = <p>Take your passwords everywhere with Firefox Lockbox. This Test Pilot experiment gives you access to all the logins you've saved to Firefox, in a secure app on your iPhone. Just log in to Lockbox with your Firefox Account, and your saved usernames and passwords will sync to the app using 256-bit encryption. So you’ll have easy access to online accounts, even on the go.</p> <p><strong>Firefox Lockbox requires a Firefox Account in order to access your previously saved logins. If you do not have an account set up, get started:<br/><a href="https://lockbox.firefox.com/faq.html">https://lockbox.firefox.com/faq.html</a></strong></p> <p><em>Want to be notified when our Android app launches? Sign up:<br/><a href="https://goo.gl/forms/ZwLIfHSGLrYcM6k83">https://goo.gl/forms/ZwLIfHSGLrYcM6k83</a></em></p>
```
Comment (flodolo) [inline locales/en-US/experiments.ftl]: We need a new ID here. I think the process is described [here](https://github.com/mozilla/testpilot/blob/master/docs/content/reference.md#localization)

**Review by flodolo — CHANGES_REQUESTED** [review]

File: locales/en-US/experiments.ftl (hunk 1 above)
Comment (devinreams) [reply locales/en-US/experiments.ftl]: >We need a new ID here.

@flodolo Thanks for reminding me! All set in the next commit 👇

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3784/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/e2e97aab9e08b3ced56f10305d1ecc083da5274d/

**Review by flodolo — APPROVED** [review]

**Pull request merged**